syntax = "proto3";

import "bakdata/corporate/v1/utils.proto";

import "google/protobuf/timestamp.proto";

package bakdata.corporate.v1;

message Institution {
  string id = 1;
  string name = 2;
  string business_category = 3;
  repeated string memberships = 4;
  google.protobuf.Timestamp initial_registration = 5;
  string register_id = 6;
  Contact contact = 7;
  Range annual_interest_expenditure = 8;
  Range interest_staff_number = 9;
  repeated string representatives = 10;
  repeated string interest_staff = 11;
  repeated string interests = 12;
  string activities_description = 13;
  repeated string clients = 14;
  repeated string grants = 15;
  repeated string donations = 16;
  string financial_report_url = 17;
  bool disclosure_required = 18;
  string code_of_conduct_url = 19;
}
